uikit/viewmodels/src/main/java/com/sygic/maps/uikit/viewmodels/common/initialization/SdkInitializationManager.kt

```diff
@@ -25,16 +25,25 @@
 package com.sygic.maps.uikit.viewmodels.common.initialization
 
 import androidx.annotation.RestrictTo
+import com.sygic.maps.uikit.views.common.utils.logError
+import com.sygic.sdk.SygicEngine
 
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
 interface SdkInitializationManager {
 
     @InitializationState
     var initializationState: Int
 
-    @FunctionalInterface
     interface Callback {
         fun onSdkInitialized()
+        fun onError(@SygicEngine.OnInitListener.InitError error: Int) {
+            val errorType = when (error) {
+                SygicEngine.OnInitListener.InitError.InternalInit -> logError("Internal init")
```

Comment:
`logError` je predsa metoda ktora priamo loguje a ty ju tu priradujes do val a potom znova volas `logError`?

Reply:
jaj, promiň, dvakrát jsem to upravoval a takto to dopadlo, to je samozřejmě překlep :) opravím